hack/update-bindata.sh

```diff
@@ -29,6 +29,8 @@ if [[ ! -d "${KUBE_ROOT}/examples" ]]; then
 	exit 1
 fi
 
+export PATH="${GOPATH}/bin:${PATH}"
```

Comment:
I removed all instances of `go get github.com/jteeuwen/go-bindata/go-bindata`, though there are still other utilities we `go get`, like `godep`, `gazel`, `goimports`, and the go `cover` command. I haven't touched those.

Like `teststale`, this is only built for the host platform, so it's installed in `$KUBE_GOPATH/bin` rather than the `_output/bin` which gets distributed. See [this comment](https://github.com/kubernetes/kubernetes/blob/3c692ebd77107dafd93248eb698adb05c6cf3ef3/hack/lib/golang.sh#L522-L529). It's gross, but I don't really feel like changing that right now.